atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnRangeSelections.java

```diff
@@ -20,9 +20,13 @@
 import com.google.common.base.Preconditions;
 import com.palantir.common.persist.Persistable;
 
-public class ColumnRangeSelections implements Serializable {
+public final class ColumnRangeSelections implements Serializable {
+    private ColumnRangeSelections() {
+        // Utility class
+    }
+
     public static ColumnRangeSelection createPrefixRange(byte[] prefix, int batchSize) {
-        byte[] startCol = Preconditions.checkNotNull(prefix).clone();
+        byte[] startCol = Preconditions.checkNotNull(prefix, "prefix cannot be null").clone();
```

Comment:
I'd prefer just disabling the check - the value of the `checkNotNull` is getting a prompt NPE in a good location. I don't know what error message we'd want to add to that - it should never happen in prod.